src/rebar_prv_packages.erl

```diff
@@ -50,17 +55,17 @@ print_packages(Pkgs) ->
                         ?CONSOLE("~s:~n    Versions: ~s~n", [Name, VsnStr])
                 end, Pkgs).
 
-sort_packages() ->
+sort_packages(T) ->
     ets:foldl(fun({package_index_version, _}, Acc) ->
                       Acc;
-                 ({Pkg, Vsns}, Acc) ->
+                 ({Pkg, [Vsns | _]}, Acc) when is_binary(Pkg) ->
```

Comment:
They aren't. But the registry has two types of package entries. One is `{{Pkg, Vsn}, [...]}` and the other (the one we want here) is `{Pkg, [...]}`.